modules/Campaigns/language/en_us.lang.php

```diff
@@ -111,23 +115,28 @@
     'LNK_NEW_EMAIL_TEMPLATE' => 'Create Email Template',
     'LNK_EMAIL_TEMPLATE_LIST' => 'View Email Templates',
     'LBL_TRACK_BUTTON_TITLE' => 'View Status',
+    'LBL_TRACK_BUTTON_KEY' => 'T',
     'LBL_TRACK_BUTTON_LABEL' => 'View Status',
     'LBL_QUEUE_BUTTON_TITLE' => 'Send Emails',
+    'LBL_QUEUE_BUTTON_KEY' => 'u',
     'LBL_QUEUE_BUTTON_LABEL' => 'Send Emails',
     'LBL_TEST_BUTTON_TITLE' => 'Send Test',
+    'LBL_TEST_BUTTON_KEY' => 'e',
     'LBL_TEST_BUTTON_LABEL' => 'Send Test',
     'LBL_COPY_AND_PASTE_CODE' => 'Or copy and paste the html below into an existing page',
     'LBL_CHARSET_NOTICE' => 'NOTE: Please make sure that the page containing the web-to-lead form has the following lines in the &lt;head&gt; section:',
 
     'LBL_TODETAIL_BUTTON_TITLE' => 'View Details',
+    'LBL_TODETAIL_BUTTON_KEY' => 'T',
     'LBL_TODETAIL_BUTTON_LABEL' => 'View Details',
 
     'LBL_DEFAULT' => 'All Target Lists',
     'LBL_MESSAGE_QUEUE_TITLE' => 'Message Queue',
+    'LBL_LOG_ENTRIES_TITLE' => 'Responses',
 
     'LBL_LOG_ENTRIES_TARGETED_TITLE' => 'Message Sent/Attempted',
-    'LBL_LOG_ENTRIES_SEND_ERROR_TITLE' => 'Bounced Messages, Other',
-    'LBL_LOG_ENTRIES_INVALID_EMAIL_TITLE' => 'Bounced Messages, Invalid Email',
+    'LBL_LOG_ENTRIES_SEND_ERROR_TITLE' => 'Bounced Messages,Other',
```

Comment:
Why removing space into a broken english?

Reply:
Thanks @horus68, should be fixed now, I'm guessing I accidentally took the majority of the old (SuiteCRM 7.5.5) language file here when I was doing the revert.